.github/actions/spell-check/expect.txt

```diff
@@ -525,6 +525,7 @@ GETPROPERTYSTOREFLAGS
 GETSCREENSAVERRUNNING
 GETSECKEY
 GETTEXTLENGTH
+gfb
```

Comment:
This is... unfortunate. It's getting picked up from a version number: `<Sdk Name="Microsoft.Build.RunVSTest" Version="1.0.319-gfb0a2ba7be" />`. It'll change over time as well.